examples/worldview/src/features/map/Map.js

```diff
@@ -175,11 +175,11 @@ export class Map extends Component {
 
     const canvasClientSize = getCanvasClientSize(resolution, viewportMinAxis);
     // canvasClientSize * scalar = previewSize
-    const scalar = scale(previewSize, canvasClientSize);
+    // const scalar = scale(previewSize, canvasClientSize);
```

Comment:
Since this only needs to be disabled for mapbox 2.5, but could still be used with 2.4 - can you treat previewSize as optional, where if it's provided then it should do this scaling and size the CSS container variable using previewSize. Otherwise (in the 2.5 case), don't scale and actually set the container size using the canvasSize variable?